<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
<meta http-equiv="x-ua-compatible" content="IE=edge"/>
<title>Test results - Class ErrorTest</title>
<link href="../css/base-style.css" rel="stylesheet" type="text/css"/>
<link href="../css/style.css" rel="stylesheet" type="text/css"/>
<script src="../js/report.js" type="text/javascript"></script>
</head>
<body>
<div id="content">
<h1>Class ErrorTest</h1>
<div class="breadcrumbs">
<a href="../index.html">all</a> &gt; 
<a href="../packages/default-package.html">default-package</a> &gt; ErrorTest</div>
<div id="summary">
<table>
<tr>
<td>
<div class="summaryGroup">
<table>
<tr>
<td>
<div class="infoBox" id="tests">
<div class="counter">2</div>
<p>tests</p>
</div>
</td>
<td>
<div class="infoBox" id="failures">
<div class="counter">1</div>
<p>failures</p>
</div>
</td>
<td>
<div class="infoBox" id="ignored">
<div class="counter">0</div>
<p>ignored</p>
</div>
</td>
<td>
<div class="infoBox" id="duration">
<div class="counter">1.519s</div>
<p>duration</p>
</div>
</td>
</tr>
</table>
</div>
</td>
<td>
<div class="infoBox failures" id="successRate">
<div class="percent">50%</div>
<p>successful</p>
</div>
</td>
</tr>
</table>
</div>
<div id="tabs">
<ul class="tabLinks">
<li>
<a href="#tab0">Failed tests</a>
</li>
<li>
<a href="#tab1">Tests</a>
</li>
<li>
<a href="#tab2">Standard output</a>
</li>
<li>
<a href="#tab3">Standard error</a>
</li>
</ul>
<div id="tab0" class="tab">
<h2>Failed tests</h2>
<div class="test">
<a name="wireMockFault"></a>
<h3 class="failures">wireMockFault</h3>
<span class="code">
<pre>org.apache.http.NoHttpResponseException: localhost:8080 failed to respond
	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:141)
	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:56)
	at org.apache.http.impl.io.AbstractMessageParser.parse(AbstractMessageParser.java:259)
	at org.apache.http.impl.DefaultBHttpClientConnection.receiveResponseHeader(DefaultBHttpClientConnection.java:163)
	at org.apache.http.impl.conn.CPoolProxy.receiveResponseHeader(CPoolProxy.java:157)
	at org.apache.http.protocol.HttpRequestExecutor.doReceiveResponse(HttpRequestExecutor.java:273)
	at org.apache.http.protocol.HttpRequestExecutor.execute(HttpRequestExecutor.java:125)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:272)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:186)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:72)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:221)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:165)
	at groovyx.net.http.HTTPBuilder.doRequest(HTTPBuilder.java:521)
	at groovyx.net.http.RESTClient.get(RESTClient.java:119)
	at groovyx.net.http.RESTClient$get.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:116)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:128)
	at ErrorTest.wireMockFault(ErrorTest.groovy:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.github.tomakehurst.wiremock.junit.WireMockRule$1.evaluate(WireMockRule.java:79)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.runTestClass(JUnitTestClassExecutor.java:110)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:58)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:38)
	at org.gradle.api.internal.tasks.testing.junit.AbstractJUnitTestClassProcessor.processTestClass(AbstractJUnitTestClassProcessor.java:62)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:33)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:94)
	at com.sun.proxy.$Proxy2.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:119)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:182)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:164)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:414)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:56)
	at java.lang.Thread.run(Thread.java:748)
</pre>
</span>
</div>
</div>
<div id="tab1" class="tab">
<h2>Tests</h2>
<table>
<thead>
<tr>
<th>Test</th>
<th>Duration</th>
<th>Result</th>
</tr>
</thead>
<tr>
<td class="success">firstTest</td>
<td class="success">0.011s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="failures">wireMockFault</td>
<td class="failures">1.508s</td>
<td class="failures">failed</td>
</tr>
</table>
</div>
<div id="tab2" class="tab">
<h2>Standard output</h2>
<span class="code">
<pre>2021-02-17 08:48:08,066 DEBUG  RESTClient - GET http://localhost:8080/fault
2021-02-17 08:48:08,108 DEBUG  headers - http-outgoing-0 &gt;&gt; GET /fault HTTP/1.1
2021-02-17 08:48:08,109 DEBUG  headers - http-outgoing-0 &gt;&gt; Accept: */*
2021-02-17 08:48:08,109 DEBUG  headers - http-outgoing-0 &gt;&gt; Host: localhost:8080
2021-02-17 08:48:08,109 DEBUG  headers - http-outgoing-0 &gt;&gt; Connection: Keep-Alive
2021-02-17 08:48:08,109 DEBUG  headers - http-outgoing-0 &gt;&gt; User-Agent: Apache-HttpClient/4.5.12 (Java/1.8.0_252)
2021-02-17 08:48:08,109 DEBUG  headers - http-outgoing-0 &gt;&gt; Accept-Encoding: gzip,deflate
2021-02-17 08:48:08,467 INFO   RetryExec - I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://localhost:8080: The target server failed to respond
2021-02-17 08:48:08,467 INFO   RetryExec - Retrying request to {}-&gt;http://localhost:8080
2021-02-17 08:48:08,468 DEBUG  headers - http-outgoing-1 &gt;&gt; GET /fault HTTP/1.1
2021-02-17 08:48:08,468 DEBUG  headers - http-outgoing-1 &gt;&gt; Accept: */*
2021-02-17 08:48:08,468 DEBUG  headers - http-outgoing-1 &gt;&gt; Host: localhost:8080
2021-02-17 08:48:08,469 DEBUG  headers - http-outgoing-1 &gt;&gt; Connection: Keep-Alive
2021-02-17 08:48:08,469 DEBUG  headers - http-outgoing-1 &gt;&gt; User-Agent: Apache-HttpClient/4.5.12 (Java/1.8.0_252)
2021-02-17 08:48:08,469 DEBUG  headers - http-outgoing-1 &gt;&gt; Accept-Encoding: gzip,deflate
2021-02-17 08:48:08,477 INFO   RetryExec - I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://localhost:8080: The target server failed to respond
2021-02-17 08:48:08,477 INFO   RetryExec - Retrying request to {}-&gt;http://localhost:8080
2021-02-17 08:48:08,478 DEBUG  headers - http-outgoing-2 &gt;&gt; GET /fault HTTP/1.1
2021-02-17 08:48:08,478 DEBUG  headers - http-outgoing-2 &gt;&gt; Accept: */*
2021-02-17 08:48:08,478 DEBUG  headers - http-outgoing-2 &gt;&gt; Host: localhost:8080
2021-02-17 08:48:08,478 DEBUG  headers - http-outgoing-2 &gt;&gt; Connection: Keep-Alive
2021-02-17 08:48:08,478 DEBUG  headers - http-outgoing-2 &gt;&gt; User-Agent: Apache-HttpClient/4.5.12 (Java/1.8.0_252)
2021-02-17 08:48:08,478 DEBUG  headers - http-outgoing-2 &gt;&gt; Accept-Encoding: gzip,deflate
2021-02-17 08:48:08,484 INFO   RetryExec - I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}-&gt;http://localhost:8080: The target server failed to respond
2021-02-17 08:48:08,485 INFO   RetryExec - Retrying request to {}-&gt;http://localhost:8080
2021-02-17 08:48:08,486 DEBUG  headers - http-outgoing-3 &gt;&gt; GET /fault HTTP/1.1
2021-02-17 08:48:08,486 DEBUG  headers - http-outgoing-3 &gt;&gt; Accept: */*
2021-02-17 08:48:08,486 DEBUG  headers - http-outgoing-3 &gt;&gt; Host: localhost:8080
2021-02-17 08:48:08,486 DEBUG  headers - http-outgoing-3 &gt;&gt; Connection: Keep-Alive
2021-02-17 08:48:08,486 DEBUG  headers - http-outgoing-3 &gt;&gt; User-Agent: Apache-HttpClient/4.5.12 (Java/1.8.0_252)
2021-02-17 08:48:08,486 DEBUG  headers - http-outgoing-3 &gt;&gt; Accept-Encoding: gzip,deflate
</pre>
</span>
</div>
<div id="tab3" class="tab">
<h2>Standard error</h2>
<span class="code">
<pre>SLF4J: Failed to load class &quot;org.slf4j.impl.StaticLoggerBinder&quot;.
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
</pre>
</span>
</div>
</div>
<div id="footer">
<p>
<div>
<label class="hidden" id="label-for-line-wrapping-toggle" for="line-wrapping-toggle">Wrap lines
<input id="line-wrapping-toggle" type="checkbox" autocomplete="off"/>
</label>
</div>Generated by 
<a href="http://www.gradle.org">Gradle 6.5</a> at Feb 17, 2021 8:48:08 AM</p>
</div>
</div>
</body>
</html>
